presto-server-rpm/pom.xml

```diff
@@ -67,110 +67,101 @@
-                                </mapping>
-                                <mapping>
-                                    <!-- This should go to just /usr/lib/presto eventually. But that needs modifying
-                                    launcher.py in airlift, to have a configurable option for install_path -->
-                                    <directory>/usr/lib/presto/lib</directory>
-                                    <sources>
-                                        <source>
-                                            <location>${project.build.directory}/${server.tar.package}/lib
-                                            </location>
-                                        </source>
-                                    </sources>
-                                </mapping>
-                                <mapping>
-                                    <directory>/usr/lib/presto/lib/plugin</directory>
-                                    <sources>
-                                        <source>
-                                            <location>
-                                                ${project.build.directory}/${server.tar.package}/plugin
-                                            </location>
-                                        </source>
-                                    </sources>
-                                </mapping>
-                                <mapping>
-                                    <directory>/etc/presto</directory>
-                                    <sources>
-                                        <source>
-                                            <location>src/main/resources/dist/config</location>
-                                        </source>
-                                    </sources>
-                                </mapping>
-                                <mapping>
-                                    <directory>/usr/shared/doc/presto</directory>
-                                    <sources>
-                                        <source>
-                                            <location>${project.build.directory}/${server.tar.package}
-                                            </location>
-                                            <!-- Just include Presto's README for now.-->
+                        <groupId>uk.co.codezen</groupId>
+                        <artifactId>redlinerpm-maven-plugin</artifactId>
+                        <version>1.0</version>
+                        <extensions>true</extensions>
+
+			            <configuration>
+                            <performCheckingForExtraFiles>false</performCheckingForExtraFiles>
+
+                            <packages>
+                                <package>
+                                    <name>presto</name>
+                                    <version>${project.version}</version>
+                                    <release>1.1</release>
```

Comment:
It updates the release number. It just changes the filename and some metadata: e.g., if building off of master, the RPM filename is presto-0.109.SNAPSHOT-1.1.x86_64.rpm.

According to the RPM spec, release is "the number of times this version of the software has been packaged." (http://www.rpm.org/max-rpm/ch-rpm-file-format.html).  It's optional, though, and it's probably reasonable not to specify the release and just rely on the version (since we don't need to worry about changes to the RPM within one version of Presto).